src/test/java/exercise30/MultiplicationTableTest.java

```diff
@@ -11,7 +11,7 @@
     @Test
 
     public void doesItWork(){
-        assertThat(multiplicationTable.timesTables().equals("0 * 0 = 0\n" +
+        assertThat(multiplicationTable.timesTables()).isEqualTo("0 * 0 = 0\n" +
```

Comment:
This was actually an incorrect test - giving a false positive result.  The fixed version doesn't pass, can you fix it?